kpi/models/asset_user_partial_permission.py

```diff
@@ -2,12 +2,87 @@
 from __future__ import annotations
 
 from collections import defaultdict
+from typing import Dict, List, Literal, Union
 
 from django.db import models
 from django.utils import timezone
 
 from kpi.constants import SUFFIX_SUBMISSIONS_PERMS
-from kpi.utils.mongo_helper import MongoHelper
+
+SimplePartialPermFilter = Dict[str, str]
```

Comment:
For the records: Even with `from __future__ import annotations` , Python 3.8 does not recognize syntax `SimplePartialPermFilter = dict[str, str]`.  It raises a `TypeError: 'type' object is not subscriptable`. 

Thus `typing.Dict, typing.List` must be imported.